components/layout_2020/flow/inline.rs

```diff
@@ -67,8 +67,9 @@ struct PartialInlineBoxFragment<'box_tree> {
     parent_nesting_level: InlineNestingLevelState<'box_tree>,
 }
 
-struct InlineFormattingContextState<'box_tree, 'cb> {
-    containing_block: &'cb ContainingBlock,
+struct InlineFormattingContextState<'box_tree, 'a> {
+    absolutely_positioned_fragments: &'a mut Vec<AbsolutelyPositionedFragment<'box_tree>>,
```

Comment:
Consider making this change separately from the implementation of `inline-block` so that its rationale is written in a commit message.

Reply:
What change? This field is only added for the because layout code for `inline-block` needs to access it.